includes/acf-meta-functions.php

```diff
@@ -4,7 +4,7 @@
  * Returns an array of "ACF only" meta for the given post_id.
  *
  * @date    9/10/18
- * @since   ACF 5.8.0
```

Comment:
These references were added intentionally since there wasn't a 5.8.0 of SCF.

Reply:
Fixed in https://github.com/WordPress/secure-custom-fields/pull/127/commits/827d2d0ecbfe08f5616765353d97c114f952d58c

Reply:
Related to this, I [opened a PR](https://github.com/WordPress/secure-custom-fields/pull/129) to add `SCF` to the annotations

Reply:
Related to this, I opened a PR to [add SCF to the annotations](https://github.com/WordPress/secure-custom-fields/pull/129).